macos/cronet.podspec

```diff
@@ -0,0 +1,22 @@
+#
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html.
+# Run `pod lib lint cronet.podspec` to validate before publishing.
+#
+Pod::Spec.new do |s|
+  s.name             = 'cronet'
+  s.version          = '0.0.6'
+  s.summary          = 'Experimental Cronet dart bindings.'
+  s.description      = <<-DESC
+  Experimental Cronet dart bindings.
+                       DESC
+  s.homepage         = 'https://pub.dev/packages/cronet'
+  s.license          = { :file => '../LICENSE' }
+  s.author           = { 'Google LLC' => 'email@example.com' }
+  s.source           = { :path => '.' }
+  s.source_files     = 'Classes/**/*'
+  s.dependency 'FlutterMacOS'
+
+  s.platform = :osx, '10.11'
+  s.pod_target_xcconfig = { 'DEFINES_MODULE' => 'YES' }
+  s.swift_version = '5.0'
```

Comment:
Missing s.vendored_libraries here.

Reply:
Oow! Now I got it. My first time using mac's build system. 😅

Reply:
I added that and is getting copied into the Framework subdirectory under the app's installation directory. But, it seems like mac is trying to find the dylib from some other path `dyld: Library not loaded: @executable_path/libcronet.86.0.4240.198.dylib`. I'm yet to figure out how to change it. I tried setting `DYLD_LIBRARY_PATH` and `LD_RUNPATH_SEARCH_PATHS`. But, didn't work.

Reply:
Try just `libcronet.86.0.4240.198.dylib` by name instead of with `@executable_path/`.

Reply:
I'm trying to load it without @executable_path. It is getting added by itself. And, this error is happening just before our app's launch. So, I guess podspec is maybe.. dynamically linking our dylibs? 
(Even if in that case, I'm unsure why it will fail to locate itself).

Reply:
We're not building cronet from source files. Can we add it to cmake file?

Reply:
It seems like a widespread issue in a few of the latest MacOS/iOS versions. Reading other related issues, I tried to use `use_frameworks!` and `use_modular_headers!` in the `Podfile`. Though some of them got lucky with this hack, I wasn't one of them (along with few others).

I'm experimenting with the issue. Meanwhile, if you get time, can try running the build on your machine maybe..? If the same error happens in your machine too?

And, is there any way to just copy the dylib and not linking it? So that it doesn't get loaded automatically and we can get the scope of loading it from Dart (by experimenting with the path).

Reply:
> We're not building cronet from source files.

`install_name_tool` can be used to investigate the "install name" for dylibs which are already compiled.
 
https://stackoverflow.com/questions/33991581/install-name-tool-to-update-a-executable-to-search-for-dylib-in-mac-os-x